src/main/java/com/jnape/palatable/lambda/adt/hlist/Tuple5.java

```diff
@@ -273,6 +273,15 @@ public _5 _5() {
         return fn.apply(_5).fmap(_3Prime -> fmap(constantly(_3Prime))).<TravB>fmap(Applicative::coerce).coerce();
     }
 
+    /**
+     * Returns a Tuple of all the elements of this Tuple except the last
```

Comment:
Let's link to the actual `Tuple` type in the javadoc, both here and in the `@return` so editors / html link correctly.